lib/ansible/modules/cron.py

```diff
@@ -701,6 +692,17 @@ def main():
             if len(old_job) > 0:
                 crontab.remove_job(name)
                 changed = True
+                if crontab.cron_file and crontab.is_empty():
```

Comment:
i woudl still keep the not `do_install` check

Reply:
You are aware, that this inside the `else` branch of `if do_install:` in line 671?

Reply:
no, had missed that